### PR TITLE
Add timestamp to "All good" message

### PR DESCRIPTION
**Pull request by vaibhavsagar**
Here's a quick first stab at the timestamp behaviour requested in #124. My setup is a bit unusual (NixOS) and I couldn't get the tests running so I don't know if/how the tests need to be updated if at all.

### CODE REVIEW

File: src/Ghcid.hs [hunk 1]
```diff
@@ -216,15 +218,15 @@ runGhcid session waiter termSize termOutput opts@Options{..} = do
 
             let updateTitle extra = unless no_title $ setTitle $
                     let f n msg = if n == 0 then "" else show n ++ " " ++ msg ++ ['s' | n > 1]
-                    in (if countErrors == 0 && countWarnings == 0 then allGoodMessage else f countErrors "error" ++
+                    in (if countErrors == 0 && countWarnings == 0 then (allGoodMessage ++ " at " ++ currTime) else f countErrors "error" ++
```
Comment (hanjoosten) [inline src/Ghcid.hs]: remove these brackets to get hlint happy 👍

Comment (vaibhavsagar) [reply src/Ghcid.hs]: Sure!

**Comment by ndmitchell** [comment]
Thanks for the patch! I'll take a look in the next day or so.

**Comment by yairchu** [comment]
Nice! I now use this :)

**Comment by ndmitchell** [comment]
Thanks! I tweaked the code to use the `time` packages `formatTime` function, and included the seconds as well, since that seemed sensible to you know if it's changing or has been frozen for the last minute. Code in 8bc299a.

**Comment by vaibhavsagar** [comment]
Thanks so much!

**Pull request merged**